springfield/firefox/templates/firefox/landing/windows.html

```diff
@@ -0,0 +1,313 @@
+
+  <section class="mzp-l-content t-releases" id="next">
+    <div class="mzp-c-emphasis-box js-animate">
+      <ul class="c-trio">
+        <li>
+          <img alt="{{ ftl('firefox-home-happy-toggle') }}" src="{{ static('img/firefox/home/toggle.svg') }}" width="90" height="54">
+          <h3>{{ feature_one_title }}</h3>
+          <p>{{ feature_one_description }}</p>
+        </li>
+        <li class="t-cursor">
+          <img alt="{{ ftl('firefox-home-multiple-cursors') }}" src="{{ static('img/firefox/home/multi.svg') }}" width="134" height="38">
+          <h3>{{ feature_two_title }}</h3>
+          <p>{{ feature_two_description }}</p>
+        </li>
+        <li>
+          <img alt="{{ ftl('firefox-home-lots-of-open') }}" src="{{ static('img/firefox/home/tabs.svg') }}" width="72" height="54">
+          <h3>{{ feature_three_title }}</h3>
+          <p>{{ feature_three_description }}</p>
+        </li>
+      </ul>
+
+      <p class="c-notes"><a class="mzp-c-cta-link" href="{{ url('firefox.notes') }}" data-cta-text="See Release Notes">{{ see_release }}</a></p>
+    </div>
+  </section>
+
+  <section class="t-highlights">
+    <h2 class="mzp-c-section-heading">{{ hightlights_title }}</h2>
+
+    <div class="t-block c-block l-reversed js-animate">
+      <div class="c-block-container">
+        <div class="c-block-body l-v-center l-h-start">
+          <h3 class="mzp-u-title-sm">{{ hightlight_one_title }}</h3>
+          <p>{{ hightlight_one_description }}</p>
+          <button id="protection-report" type="button" class="mzp-c-cta-link">{{ ftl('firefox-desktop-download-see-your-report') }}</button>
+        </div>
+        <div class="c-block-media l-v-end l-h-end l-media-constrain-on-sm">
+        <img alt="{{ ftl('firefox-home-shield-and-hand') }}" class="c-block-media-img" src="{{ static('img/firefox/home/block.svg') }}" width="220">
+        </div>
+      </div>
+    </div>
+
+    <div class="t-think c-block js-animate">
+      <div class="c-block-container">
+        <div class="c-block-body l-v-center l-h-end">
+          <h3 class="mzp-u-title-sm">{{  hightlight_two_title }}</h3>
+          <p>{{ hightlight_two_description | safe }}</p>
+        </div>
+        <div class="c-block-media l-v-center l-h-start l-media-constrain-on-sm">
+        <img alt="{{ ftl('firefox-home-a-human-brain') }}" class="c-block-media-img" src="{{ static('img/firefox/home/think.svg') }}" width="260">
+        </div>
+      </div>
+    </div>
+
+    <div class="t-devices c-block l-reversed js-animate">
+      <div class="c-block-container">
+        <div class="c-block-body l-v-center">
+          <h3 class="mzp-u-title-sm">{{ hightlight_three_title }}</h3>
+          <p>{{ hightlight_three_description }}</p>
+          <div class="mobile-download-buttons-wrapper">
+            {% block mobile_secondary_cta %}
+              <ul class="mobile-download-buttons">
+                <li class="android">
+                  {{ google_play_button() }}
```

Comment:
also accepts `href` param

the default currently shows no indication of this campaign
<img width="749" height="26" alt="Screenshot 2025-10-06 at 2 55 46 PM" src="https://github.com/user-attachments/assets/351dd71a-06f9-48a7-85db-f4112682c9bc" />